filesystem/Upload.php

```diff
@@ -208,6 +208,7 @@ public function load($tmpFile, $folderPath = false) {
 			$this->file->ParentID = $parentFolder ? $parentFolder->ID : 0;
 			// This is to prevent it from trying to rename the file
 			$this->file->Name = basename($relativeFilePath);
+			$this->file->Title = preg_replace("/{$fileSuffix}$/", '', $tmpFile['name']);
```

Comment:
@dhensby
Thanks for `preg_quote`

`FileNameFilter.default_use_transliterator = false`  isn't a solution. 
Look here.
https://github.com/silverstripe/silverstripe-framework/pull/6621#issuecomment-279725738

Yes, $fileSuffix is already transliterated by this point and situation you've described is possible.

For now I haven't graceful way how to reach a goal and not break something else.

So, what will we do with this PR?
Maybe we'll close it and I'll create new PR next time?
Or can I commit to current PR?